test/BUILD

```diff
@@ -273,6 +273,21 @@ plz_e2e_test(
     expected_failure = True,
 )
 
+# Test add_data functionality
+genrule(
+    name = "_add_data_gen",
+    cmd = "echo hello > _add_data_gen.txt",
+    pre_build = lambda name: add_data(name, "add_data_gen.txt"),
+)
+
+gentest(
+    name = "_add_data_test",
+    data = [":_add_data_gen"],
+    labels = ["manual"],
```

Comment:
This marks the test as a manual test. This means it won't get run on the CI servers. Not sure that's what we want.